networktable.c

```diff
@@ -76,15 +92,16 @@ void parse_arp_cache(void)
 	// Read ARP cache line by line
 	while(getline(&linebuffer, &linebuffersize, arpfp) != -1)
 	{
-		int num = sscanf(linebuffer, "%99s 0x%x 0x%x %99s %99s %99s\n",
-		                 ip, &type, &flags, hwaddr, mask, iface);
+		int num = sscanf(linebuffer, "%99s dev %99s lladdr %99s %99s",
```

Comment:
On my machine, there is an extra word between the MAC address and the status: `router`. This may only appear on special neighbors, like the router, but with this code `status` would be set to `router` instead of `REACHABLE`.

Reply:
I haven't seen such a `router` line before, can you post an (obfuscated) example?

Reply:
```
dead:beef:dead:beef:dead:beef dev wlp3s0 lladdr de:ad:be:ef:de:ad router REACHABLE
```